Content.Server/Anomaly/Components/ChasingAnomalyComponent.cs

```diff
@@ -0,0 +1,51 @@
+using Content.Server.Anomaly.Effects;
+
+namespace Content.Server.Anomaly.Components;
+
+
+/// <summary>
+/// This component allows the anomaly to chase a random instance of the selected type component within a radius.
+/// </summary>
+[RegisterComponent, Access(typeof(ChasingAnomalySystem))]
+public sealed partial class ChasingAnomalyComponent : Component
+{
+    /// <summary>
+    /// The maximum radius in which the anomaly chooses the target component to follow
+    /// scales with severity
+    /// </summary>
+    [DataField("maxChaseRadius")]
```

Comment:
@TheShuEd no dont remove viewvariables remove the name inside DataField
[DataField, ViewVariables(VVAccess.ReadWrite)]